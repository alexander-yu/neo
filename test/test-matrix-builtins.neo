void main() {
    create matrix<int> a = [[1, 2], [3, 4], [5, 6]];
    println(insert(a, 1, [[7, 8]]));
    println(delete(a, 0));
    println(append(a, [[9, 10]]));
}