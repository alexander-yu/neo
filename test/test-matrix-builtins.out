[[1, 2],
 [7, 8],
 [3, 4],
 [5, 6]]
[[3, 4],
 [5, 6]]
[[1, 2],
 [3, 4],
 [5, 6],
 [9, 10]]
